### ISSUE TITLE
Add an option to specify a tarball subdirectory when using wsl --import

### ISSUE THREAD

**Issue by Vaporeon** [microsoft/WSL#9187]
**Is your feature request related to a problem? Please describe.**
Many Linux distributions provide bootstrap tarballs alongside full installation media.
One popular example of these is Arch Linux.

The bootstrap media provided by upstream Arch Linux is *close* to what `wsl --import` can use for input, the only difference being that the root filesystem in the bootstrap tarball is located in the `root.x86_64` directory.

bootstrap tarballs can be obtained by clicking one of the mirrors [here](https://archlinux.org/download/)

**Describe the solution you'd like**
It would be handy to have an option that lets you tell `wsl --import` where in the tar the root file system is located.
If we assume the new option is called `--prefix`, one could run `wsl --import <tar> --prefix root.x86_64` to get a running Arch Linux instance from original upstream media.

**Describe alternatives you've considered**
The current way to instal a distro like Arch Linux in the WSL is to use a separate Linux instance to re-create the tarball as such that the root filesystem is in the root of the tar archive.

This means that to do this on Windows you will need to install some other Linux distribution just to run the correct `tar` command. This must be done using Linux so permissions are preserved correctly.

**Comment by Biswa96** [comment]
> The current way to install a distro like Arch Linux in the WSL is to use a separate Linux instance

I agree with the issue. But there is a way to install the Arch Linux bootstrap tarball which does not require a separate Linux instance. I am suggesting the steps just as a workaround, not as a solution.

1. Install the Arch Linux bootstrap media as a WSL1 distribution.
2. Open the install directory in File Explorer. Move all the contents from `root.x86_64` directory to that install directory. Try not to copy-paste from keyboard. Instead, select all bin, etc, usr etc. directories and drag-drop in the folder above.
3. Convert the distribution from WSL1 to WSL2. Tada :)

The procedure takes advantage of the fact that moving files in step 2 does not change the NTFS extended attributes i.e. the Linux files' owner, uid, gid, timestamp etc.
